src/home/NotificationBox.tsx

```diff
@@ -224,7 +230,7 @@ export function useSimpleActions() {
         })
       }
 
-      if (!isSupercharging && !dismissedStartSupercharging) {
+      if (!isSupercharging && !restrictSuperchargeForClaimOnly && !dismissedStartSupercharging) {
```

Comment:
We have a number of unit tests for supercharge notifications.
I wonder if we want to augment them for that case?